lib/elastic_sync/index.ex

```diff
@@ -28,8 +28,8 @@ defmodule ElasticSync.Index do
     |> HTTP.post
   end
 
-  def transition(name, fun) do
-    transition(name, get_new_alias_name(name), fun)
+  def transition(name, fun, config \\ []) do
```

Comment:
we should be consistent with the type we're expecting for `config`. Is it a keyword list or a map? Technically tirexs will accept both, but I'd actually prefer we consistently default to a map.